components/userPages/Login/LogInForm.js

```diff
@@ -0,0 +1,42 @@
+import { useState, useContext } from "react";
+import { LoginContext } from "../../contexts/loginContext.js";
+
+import Input from "./UserInput";
+
+export default function UserLoginForm() {
+  const [userName, setUserName] = useState("");
+  const [password, setPassword] = useState("");
+  const { setUserObj } = useContext(LoginContext);
+
+  async function handleSubmit(e) {
+    e.preventDefault();
+    const data = { userName, password };
+    const res = await fetch("/api/users", {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(data),
+    });
+    const Obj = await res.json();
+
+    if (!Obj.err) {
+      setUserObj({ ...Obj, logedIn: true });
+    } else {
+      alert(Obj.message)
+    }
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <Input _label={"User Name: "} _setState={setUserName} />
```

Comment:
Jag föreslår att en användare loggar in med sin e-post istället. Dels för att undvika kollisioner med användare som delar samma namn. E-post adresser är även unikt över hela världen! 🚀

Reply:
> Det kan jag hålla med om, fixar det! Är det något som vi vill ha gjort före en merge nu eller tar vi det efteråt?

Det kan du göra sen om du känner för det 👍